sql/dd/impl/types/object_table_impl.h

```diff
@@ -125,6 +127,8 @@ class Object_table_impl : virtual public Object_table {
 
   virtual void set_encrypted();
```

Comment:
Although the person who reviews this code should understand what actual vs target table means, it is better if there is atleast 'some' comment on function (along with description in commit message)

Reply:
> I tried to reproduce the above crash in 8.0.15 and I couldn't. May be specific to patch or specific to 8.0.16. Will check

There is new bug with 8.0.16. default_table_encryption=ON at bootstrap doesn't encrypt mysql.ibd.  I fixed this (wrote patch and I have to submit PR)

8.0.16 doesn't have the above issue. So this crash is specific to upgrade of encrypted mysql.ibd from 8.0.15 to 8.0.16

Reply:
Fixed as part of PS-5925 and PS-5926 (the blockers for this PR)